src/components/Watchlists/NewWatchlistForm.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import cx from 'classnames'
+import { Button, Toggle, Input } from '@santiment-network/ui'
+import * as actions from './../../actions/types'
+import styles from './NewWatchlistForm.module.scss'
+
+const MAX_NAME_LENGTH = 50
+const ERROR_MESSAGE =
```

Comment:
max name length in mock-up: 25

Reply:
![image](https://user-images.githubusercontent.com/24521041/56348723-c30c5280-61cf-11e9-8791-c6ea16506360.png)
other colors here